### PR TITLE
[blogger] Cleanups:

### PR DESCRIPTION
**Pull request by Quix0r**
- used `use Friendica\App;`
- added type-hints for `App` and `array` where applyable
- removed extra space
- added other spaces

### CODE REVIEW

File: blogger/blogger.php [hunk 1]
```diff
@@ -14,35 +15,35 @@
...
-	Addon::unregisterHook('addon_settings_post',  'addon/blogger/blogger.php', 'blogger_settings_post');
+	Addon::unregisterHook('post_local_end',      'addon/blogger/blogger.php', 'blogger_send');
+	Addon::unregisterHook('addon_settings',      'addon/blogger/blogger.php', 'blogger_settings');
+	Addon::unregisterHook('addon_settings_post', 'addon/blogger/blogger.php', 'blogger_settings_post');
 }
 
 
-function blogger_jot_nets(&$a, &$b)
-{
+function blogger_jot_nets(App $a, &$b) {
```
Comment (MrPetovan) [inline blogger/blogger.php]: New line please.

File: blogger/blogger.php [hunk 2]
```diff
@@ -115,18 +116,18 @@ function blogger_settings(&$a, &$s)
 }
 
 
-function blogger_settings_post(&$a, &$b)
+function blogger_settings_post(App $a, array &$b)
 {
-	if (x($_POST, 'blogger-submit')) {
-		PConfig::set(local_user(), 'blogger', 'post', intval($_POST['blogger']));
+	if (isset($_POST['blogger-submit'])) {
```
Comment (MrPetovan) [inline blogger/blogger.php]: The exact equivalent to `x($_POST, 'blogger-submit')` is `!empty($_POST['blogger-submit'])`.

Comment (Quix0r) [reply blogger/blogger.php]: Done.

Comment (MrPetovan) [reply blogger/blogger.php]: Aren't you supposed to sleep? 😅

**Pull request merged**